baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/PreferSafeLoggingPreconditions.java

```diff
@@ -64,7 +63,7 @@
                             .onClass("org.apache.commons.lang3.Validate")
                             .withNameMatching(Pattern.compile("isTrue|notNull|validState")));
 
-    private static final Map<String, String> TRANSLATIONS_TO_LOGSAFE_PRECONDITIONS_METHODS = ImmutableMap.of(
+    private static final ImmutableMap<String, String> TRANSLATIONS_TO_LOGSAFE_PRECONDITIONS_METHODS = ImmutableMap.of(
```

Comment:
Unrelated change, but error prone was warning against this when running tests locally - kind of meta..